### ISSUE TITLE
in a java app nvda does not announce the hover menu 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13043]
### Steps to reproduce:
in a java application press alt to access the menu bar of this application 
### Actual behavior:
nvda repeats the name of the previously focused button 
### Expected behavior:

I would like to explain better what happens.
This is a java application developed internally in the organization where I work.
I believe this problem can be represented in other java applications, despite not having tested it, so I opened this call.
I will explain what I did in this application.
I walked with tabe through the application's interface and focused on a button, which in Portuguese calls resolve pending issues.
so far so good.
I have now pressed the alt key.
the focused menu is now called system.
However nvda did not announce the system.
Nvda repeated the name of the button previously in focus which was to resolve pending issues.
I know that the system menu is in focus because if I walk with the arrow below the nvda read the items from the system menu.
what caught my attention too, is that in the log the pressing of the alt key did not appear, in the log it is as if the key had not been pressed. 
nvda should announce the name of the open menu. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
alfa 24184
#### Windows version:
11 22.000.282

#### Name and version of other software in use when reproducing the issue:
java app Java(TM) Platform SE 8 version 8.0.3110.11
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
[javamenu.txt](https://github.com/nvaccess/nvda/files/7500478/javamenu.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still having this issue with NVDA 2024.4.1?

**Comment by fernando-jose-silva** [comment]
Thank you very much, this has been resolved.Enviado do meu iPhoneEm 27 de nov. de 2024, à(s) 20:42, Adriani90 ***@***.***> escreveu:﻿
@fernando-jose-silva are you still having this issue with NVDA 2024.4.1?

**Comment by Adriani90** [comment]
Thanks for the update, closing as works for me.
